### PR TITLE
updated noobaa-operator to 2.1 + vendor

### PR DESCRIPTION
**Pull request by dannyzaken**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,7 +11,7 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
 	github.com/go-openapi/spec v0.19.2
-	github.com/noobaa/noobaa-operator/v2 v2.0.8
+	github.com/noobaa/noobaa-operator/v2 v2.0.6-0.20200219114615-8ae457e831e4
```
Comment (umangachapagain) [inline go.mod]: Is this noobaa-operator 2.1 ?

File: go.mod [hunk 2]
```diff
@@ -22,13 +22,12 @@ require (
...
 	github.com/stretchr/testify v1.4.0
 	go.uber.org/zap v1.13.0
-	google.golang.org/appengine v1.6.1 // indirect
 	k8s.io/api v0.0.0-20191005115622-2e41325d9e4b
 	k8s.io/apiextensions-apiserver v0.0.0-20190820104113-47893d27d7f7
 	k8s.io/apimachinery v0.0.0-20191005115455-e71eb83a557c
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	k8s.io/cloud-provider v0.0.0-20191114015453-79225fba1e26 // indirect
-	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf
+	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
```
Comment (umangachapagain) [inline go.mod]: Is this related to nooba changes?

File: vendor/modules.txt [hunk 3]
```diff
@@ -191,6 +191,9 @@ github.com/prometheus/common/model
 github.com/prometheus/procfs
 github.com/prometheus/procfs/internal/fs
 # github.com/rook/rook v1.2.4
+github.com/rook/rook/pkg/apis/cassandra.rook.io
+github.com/rook/rook/pkg/apis/cassandra.rook.io/v1alpha1
+github.com/rook/rook/pkg/apis/ceph.rook.io
```
Comment (umangachapagain) [inline vendor/modules.txt]: Doesn't seem related to this PR.

File: go.mod (hunk 1 above)
Comment (dannyzaken) [reply go.mod]: 2.1 is not released yet. I point to the 2.1 branch for now and will update once it will be released

File: go.mod (hunk 2 above)
Comment (dannyzaken) [reply go.mod]: yes. it was updated with `make deps-update`

File: vendor/modules.txt (hunk 3 above)
Comment (dannyzaken) [reply vendor/modules.txt]: this was updated after `make deps-update` (`go mod vendor`) probably something that was updated in a different change.
I don't think I have control over it

**Comment by dannyzaken** [comment]
@umangachapagain I've split to 2 commits. can you review it again?

**Comment by umangachapagain** [comment]
/ok-to-test

Do we need this for OCS 4.3?

**Comment by dannyzaken** [comment]
@umangachapagain yes we do
